File: Python/List-2/has22.py
"""
Given an array of ints, return True if the array contains a 2 next to a 2 somewhere.


has22([1, 2, 2]) → True
has22([1, 2, 1, 2]) → False
has22([2, 1, 2]) → False

@author unobatbayar
"""
def has22(nums):
  for x in range(0, len(nums)-1):
    if nums[x] == 2 and nums[x+1] == 2:
      return True
  return False

File: Java/Array-2/has22.java
/**
 * Given an array of ints, return true if the array contains a 2 next to a 2 somewhere.

has22([1, 2, 2]) → true
has22([1, 2, 1, 2]) → false
has22([2, 1, 2]) → false

@author unobatbayar
 */

public boolean has22(int[] nums) {
    for(int i = 0; i<nums.length; i++){
      if(nums[i] == 2){
        if(i<nums.length-1 && nums[i+1] == 2) return true;  
        if(i > 0 && nums[i-1] == 2) return true;  
      }
    }
    return false;
  }
  